src/app/components/HoldConfirmation/HoldConfirmation.jsx

```diff
@@ -180,81 +181,83 @@ class HoldConfirmation extends React.Component {
     }
 
     return (
-      <main id="mainContent" className="main-page">
-        <div className="nypl-request-page-header">
-          <div className="row">
-            <div className="nypl-full-width-wrapper">
-              <div className="nypl-column-full">
-                <Breadcrumbs
-                  query={this.props.location.query.searchKeywords}
-                  type="confirmation"
-                  bibUrl={`/bib/${bibId}`}
-                  itemUrl={`/hold/request/${bibId}-${itemId}`}
-                  edd={pickupLocation === 'edd'}
-                />
-                <h2>{appConfig.displayTitle}</h2>
+      <DocumentTitle title={`${title} | Shared Collection Catalog | NYPL`}>
```

Comment:
According to #676 , this should be `Request Confirmation | Shared Collection Catalog | NYPL`.